app/assets/stylesheets/govuk_publishing_components/components/_reorderable-list.scss

```diff
@@ -0,0 +1,120 @@
+.gem-c-reorderable-list {
+  @include govuk-font(19, bold);
+
+  list-style-type: none;
+  margin-bottom: govuk-spacing(6);
+  margin-top: 0;
+  padding-left: 0;
+  position: relative;
+
+  .govuk-form-group {
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-reorderable-list__item {
+  margin-bottom: govuk-spacing(3);
+  border: 1px solid $govuk-border-colour;
+  padding: govuk-spacing(3);
+}
+
+.gem-c-reorderable-list__item--chosen {
+  background-color: govuk-colour('light-grey');
+  outline: 2px dotted $govuk-border-colour;
+}
+
+.gem-c-reorderable-list__item--drag {
+  background-color: govuk-colour('white');
+  list-style-type: none;
+
+  .gem-c-reorderable-list__actions {
+    visibility: hidden;
+  }
+}
+
+.gem-c-reorderable-list__wrapper {
+  display: block;
+
+  @include govuk-media-query($from: desktop) {
+    display: inline-flex;
+    width: 100%;
+  }
+}
+
+.gem-c-reorderable-list__content,
+.gem-c-reorderable-list__actions {
+  margin-bottom: govuk-spacing(2);
```

Comment:
I suspect we don't need this additional spacing anymore. I was previously being used for vertical spacing as there was no padding on the item itself. With the current design the padding in `gem-c-reorderable-list__item` ensures equal spacing. We might, however, need space between the content and the action on mobile, so I'll make a suggestion there.